client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -531,6 +532,12 @@ function getFallbackDestination( {
 		return `/checkout/thank-you/features/${ feature }/${ siteSlug }/${ receiptIdOrPlaceholder }`;
 	}
 
+	const is100YearPlanProduct = cart?.products?.some( is100Year );
+	if ( is100YearPlanProduct ) {
+		debug( 'site with 100 year plan' );
+		return `/checkout/100-year/thank-you/${ siteSlug }/${ receiptIdOrPlaceholder }`;
```

Comment:
All the outputs of this function are covered by tests (see the top of this file for info). Is it possible to add a test for this case?